src/annotator/anchoring/pdf.ts

```diff
@@ -259,6 +259,27 @@ function isSpace(char: string) {
 
 const isNotSpace = (char: string) => !isSpace(char);
 
+/**
+ * Determines if provided text layer is done rendering.
+ * It works on older PDF.js versions which expose a public renderingDone prop,
```

Comment:
I would backtick-quote `renderingDone` in a context like this to clarify that it is an identifier.